### DISCUSSION TITLE
Domoticz Setup (Instructions not complete)

### DISCUSSION THREAD

**Discussion by NsinghP** [arendst/Tasmota#12351] [Q&A] [answered]
Hello, 

I was trying to use Domoticz app to control the lights attached to my tasmota device.  I followed all the directions on how to configure at https://tasmota.github.io/docs/Domoticz/ but then when I came to the part which says "Tasmota Configuration" I lost the plot because the webpage says:

 "[The sections below don't look like they are needed any longer. The in topic and out topic entry areas don't appear to be in the Domoticz configure section - at least they are not there in mine and mine is working 17/03/2018]"

Look like I don't need to do anything special on my tasmota device and it should work. 

Needless to say I can't control my device from the Domoticz on my PC.

 I did enter "36" in Idx1 setting in tasmota which is Idx number assigned to the virtual switch in Domoticz. I also entered Hivemq.cloud settings and kept TLS version  settings at default tlsv1 in Domoticz.

at this point I think either the Domoticz doesn't work with hivemq.cloud and needs a local broker or I am missing some configuration settings in my tasmota device.

So my question: Is it possible to control my device with domoticz without using a local in house Mqtt broker?

Here are some pictures:

![Screenshot (168)](https://user-images.githubusercontent.com/69470381/121718547-0b99cf00-cb00-11eb-9117-e39be6836c30.png)
![Screenshot (169)](https://user-images.githubusercontent.com/69470381/121718588-15bbcd80-cb00-11eb-9a8a-884c5d496eb0.png)
![Screenshot (170)](https://user-images.githubusercontent.com/69470381/121718629-210ef900-cb00-11eb-915b-657bd9bdb00a.png)

Regards,

Navi

**Comment by sfromis** [comment, marked as answer]
I'd suggest forgetting about Domoticz details till you've verified the basic MQTT setup to be fully working, as in messages flowing both ways. Only after that, see what happens when you set up for Domoticz both ends. As long as your MQTT setup is working, it should make no difference if it is local or not.

My experience with Domoticz and Hivemq is zero.

Actually, I might instead start making it work locally with a good broker like Mosquitto, and after that see how bad it would be to make it work with Hivemq.

**Reply by NsinghP** [reply]
Thanks, I think MQTT setup is fully working. I can do everything that I was doing from web console before.

**Reply by NsinghP** [reply]
Just a followup---- Hivemq.cloud says that in basic version they use TLS v1.2, It seems to be a problem on Domoticz app side as it is not able to connect to Hivemq.cloud using TLS. 

I am getting the following error in Domoticz app: 
2021-06-17 11:40:11.420 Status: Hazel Test Module1: MQTT: Connecting to 9d...........87.s1.eu.hivemq.cloud:8883
2021-06-17 11:40:12.908 Error: Hazel Test Module1: MQTT: disconnected, restarting (rc=7)

On my MQTT explorer I can see the device listed under Domoticz so tasmota device is communicating and should work with local broker. 
![Screenshot (188)](https://user-images.githubusercontent.com/69470381/122342878-b9730680-cf62-11eb-8951-70f984dad127.png)
